util/import.ts

```diff
@@ -1,11 +1,50 @@
 import { TestCaseInfo } from '../state/atoms/patientTestCase';
 
-export function bundleToTestCase(bundle: fhir4.Bundle): TestCaseInfo {
+export function bundleToTestCase(bundle: fhir4.Bundle, populationGroupCodes: string[]): TestCaseInfo {
```

Comment:
Would recommend a function comment on this, especially since it's grown in scope.